.ci/run.sh

```diff
@@ -24,6 +24,12 @@ x-pack-ciGroup*)
   export CI_GROUP="${JOB##x-pack-ciGroup}"
   ./test/scripts/jenkins_xpack_ci_group.sh
   ;;
+firefox-intake*)
```

Comment:
I think I'd prefer to use `firefox-smoke` naming here, so it's not confused with the intake jobs